.templates/openstack/properties.yml

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  shield:
+    daemon_public_key: (( vault "secret/dc01/proto/shield/keys/core:public" ))
+
+properties:
+  prometheus:
+    rule_files: 
+    - /var/vcap/packages/bosh_alerts/*.alerts
+    - /var/vcap/packages/cloudfoundry_alerts/*.alerts
+    scrape_configs:
+    - (( append ))
+    - job_name: bosh_graphite
+      static_configs:
+      - targets: (( grab meta.graphite_exporter_targets ))
```

Comment:
Where are those `meta` properties (`graphite_exporter_targets`, `firehose_exporter_targets`, `cf_exporter_targets`) defined? If you want add them here, you should define a `meta` at the top of this file and add a `param` for each one.